src/mocks/index.ts

```diff
@@ -1 +1,2 @@
-export * from './posts';
+export * from "./posts";
+export * from "./DataDemo";
```

Comment:
```suggestion
export * from "./users";
```
UpperCamelCase o PascalCase deberiamos usarlo solo para componentes